### PR TITLE
Publish December RubyGems Update

### PR DESCRIPTION
**Pull request by lauragift21**
RubyGems monthly update roundup for December 2023

### CODE REVIEW

File: _posts/2024-01-08-december-rubygems-updates.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Additionally, [Bundler](https://bundler.io) introduced the `bundle config set version` feature. This allows users to explicitly select the Bundler version they wish to use, adding a layer of customization. It also provides the option to override the lockfile version by setting it to `system` ([#6817](https://github.com/rubygems/rubygems/pull/6817)).
+
+Some other improvements that landed into our repo this month that are NOT included in the above releases are:
+
+- The documentation process has been upgraded, now utilizing  [nronn](https://github.com/n-ronn/nronn) for generation ([#7227](https://github.com/rubygems/rubygems/pull/7227)).
+- Used `Minitest::TestTask` in a template file for minitest streamlining the testing process.([#7234](https://github.com/rubygems/rubygems/pull/7234)).
+- Avoid some allocations when evaluating ruby Gemfile DSL ([#7251](https://github.com/rubygems/rubygems/pull/7251)).
+- Improved install advice when some gems are not found ([#7265](https://github.com/rubygems/rubygems/pull/7265)).
+- A fix was implemented for the `bundler test` on the Ruby package ([#7298](https://github.com/rubygems/rubygems/pull/7298)).
+
```
Comment (martinemde) [inline _posts/2024-01-08-december-rubygems-updates.md:27]: A few more I think are noteworthy enough to include.
```suggestion
- `bundle lock` always touch the lockfile (even when nothing changed) to improve Make-style compatibility ([#7220](https://github.com/rubygems/rubygems/pull/7220)
- Improve rubygems and Bundler CI detection ([#7205](https://github.com/rubygems/rubygems/pull/7205)
- Stream output from ext builds when `--verbose` ([#7240](https://github.com/rubygems/rubygems/pull/7240))
- Allow bundle pristine to run in parallel, resulting in a remarkable speed improvement ([#6927](https://github.com/rubygems/rubygems/pull/6927)
- Ensure gem install respects system umask ([#7300](https://github.com/rubygems/rubygems/pull/7300))

```

File: _posts/2024-01-08-december-rubygems-updates.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+## RubyGems News
+
+This month in RubyGems, we released the following version of RubyGems [3.5.0](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#350--2023-12-15), [3.5.1](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#351--2023-12-15), [3.5.2](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#352--2023-12-21), [3.5.3](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#353--2023-12-22) and Bundler [2.5.0](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#250-december-15-2023), [2.5.1](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#251-december-15-2023), [2.5.2](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#252-december-21-2023), [2.5.3](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#253-december-22-2023). These updates mark significant milestones in our continuous effort to enhance the Rubygems development experience.
+
```
Comment (martinemde) [inline _posts/2024-01-08-december-rubygems-updates.md:13]: I suggest bringing up the ruby version support changes

```suggestion

With the yearly minor version release of rubygems 3.5 and bundler 2.5, we have dropped support for Ruby versions less than 3.0.0. This enables the usage of more modern features of the Ruby language, improving efficiency, performance and memory usage as well as reducing our test runtimes and removing the more error prone Ruby 2.6 and 2.7 tests. We spent time this month modernizing the codebase to take advantage of this change.

```

**Review by martinemde — APPROVED** [review]
I added a few commits I thought were also worth mentioning. This was a big release month so there's a lot to talk about.

**Comment by lauragift21** [comment]
> I added a few commits I thought were also worth mentioning. This was a big release month so there's a lot to talk about.

Thanks @martinemde!

**Review by deivid-rodriguez — APPROVED** [review]
Thanks so much @martinemde, great summary!

**Pull request merged**